HW Assignment #1/Portfolio.html

```diff
@@ -0,0 +1,27 @@
+<center>
+	<h1>
+		<a href="About_Me.html"> ABOUT ME</a> | PORTFOLIO
+	</h1>
+
+	<h2>TASNEEM KARIM
+	</h2>
+</center>	
+
+
+<h2>Experience
+</h2>
+				
+I recently graduated from a Front-End Web Development course at General Assemblly where I learned HTML, CSS. JavaScript, and jQuery.  During the course, I dicovered that development has a lot in common with drafting agreements.  Concepts such as version control, structure, clean formatting, and the uutilizind and customizing pre-existing language play a great part in both disciplines.
+
+
+<h3>PORTFOLIO
+</h3>
+
+<ul>
+	<li>
+		<a href=> Link#1 </a>
+	</li>
+	<li>
+		<a href=> Link#2 </a>
```

Comment:
Be more mindful about indentation. I like to run my code through an HTML formatter to test this but still review it to make sure everything is in place: http://codebeautify.org/htmlviewer/